src/main/java/org/example/workerapi/exception/EmployeeNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package org.example.workerapi.exception;
+
+public class EmployeeNotFoundException extends RuntimeException {
+    public EmployeeNotFoundException(String message) {
+        super(message);
+    }
```

Comment:
Gali ir kaip nors įdomiau čia parašyti, ne tik perduoti message, pvz:

```suggestion
    public EmployeeNotFoundException(String criteria) {
        super("Employee not found by: " + criteria);
    }
```

ir tuomet:

```
throw new EmployeeNotFound("id=" +  id);
```

```
throw new EmployeeNotFound("name=" +  name);
```

ir t.t.

Kas galiausiai throwintų errorą:

"Employee not found by: id=5"
"Employee not found by: name=Marius"

ir nereikėtų visur to "Employee not found" teksto kartoti